src/nodes/ImageView.vue

```diff
@@ -35,6 +35,28 @@
 						type="text"
 						:value="alt"
 						@keyup.enter="updateAlt()">
+					<div
```

Comment:
Would be great if we could switch to use https://www.npmjs.com/package/vue-material-design-icons directly here, so you could just import the icon from there and then we can easily switch to the button component later.